pages/blog/2022-summary.md

```diff
@@ -0,0 +1,252 @@
+---
+title: "State of AsyncAPI Initiative 2022 in Numbers"
+date: 2023-02-24T06:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+cover: /img/posts/2022-summary/cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Dev Comm Keeper
+excerpt: 'tl;dr We do not see any negative trends indicating that the interest in the project got stale and that the community is not growing anymore.'
+featured: true
+---
+
+It is a good practice to sometimes stop and turn back. Sometimes you need to see what you left behind, what successes made you go that far, and what things you missed and never had time to enjoy or explore. The more often you do it, the better your decisions may be in the future. It is essential to do it at least once a year in open source.
+
+Omg, I don't recall when I last wrote something so clever. At least, I think it sounds clever :smiley:
+
+This article will take you on a journey to the past to see what happened in 2022 at AsyncAPI Initiative.
+
+If you are interested in previous project updates [check other `Project Status` posts](https://www.asyncapi.com/blog?tags=Project+Status).
+
+## Slack
+
+At the end of 2022, we had **3114** members in the AsyncAPI workspace. But what does it really mean? Don't get me wrong, I like when the numbers are high, especially if I see them in my bank account. 
+
+The thing is that some accounts might belong to people that joined three years ago but don't even remember that they had an account in the AsyncAPI workspace. It would be good to check how this number relates to people's activity.
+
+Also important is to understand how come we have **3114** members. Did all of them join in 2016? And then we had no new joiners since then? That would be a disaster and a clear indication we are doing something wrong :sweat_smile:
```

Comment:
```suggestion
Another important item to understand is how come we have **3114** members. Did all of them join in 2016? Have we had no new people join since then? That would be a disaster and a clear indication we are doing something wrong. :sweat_smile:
```